### PR TITLE
README first session

### PR DESCRIPTION
**Pull request by hpenedones**
Changing the description of the inductiva package to be more precise and highlight its advantages.

### CODE REVIEW

**Review by cristianacarpinteiro — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -3,11 +3,19 @@
...
+# Inductiva API Python client
 
-**Inductiva API** provides open-source physical simulation from your laptop. With no configuration headaches, users can easily scale simulations to the next level with one line of code.
+Inductiva is a Python package designed for executing large-scale simulations of physical systems directly in the cloud.
 
-Whether you want to simulate pre-build scenarios to solve scientific/engineering problems or you are a power user of a specific open-source simulator, **Inductiva API** is here for you.
+This offers several distinct advantages:
+
+* Eliminates the need to install and manage complex simulation software along with their library dependencies.
+* Effortlessly run hundreds or even thousands of simulations concurrently.
```
Comment (sarmento) [inline README.md:13]: Allows running hundreds or even thousands of simulations concurrently, with no additional effort.

Comment (sarmento) [reply README.md:13]: I prefer something like:

Reduces the effort in managing dedicated hardware resources

File: README.md [hunk 2]
```diff
@@ -3,11 +3,19 @@
...
-# Large scale simulations made simple
+# Inductiva API Python client
 
-**Inductiva API** provides open-source physical simulation from your laptop. With no configuration headaches, users can easily scale simulations to the next level with one line of code.
+Inductiva is a Python package designed for executing large-scale simulations of physical systems directly in the cloud.
 
-Whether you want to simulate pre-build scenarios to solve scientific/engineering problems or you are a power user of a specific open-source simulator, **Inductiva API** is here for you.
+This offers several distinct advantages:
+
+* Eliminates the need to install and manage complex simulation software along with their library dependencies.
```
Comment (sarmento) [inline README.md:12]: Eliminates the need to install and manage complex simulation software and corresponding dependencies.

File: README.md [hunk 3]
```diff
@@ -3,11 +3,19 @@
...
+* Effortlessly run hundreds or even thousands of simulations concurrently.
+* There's no requirement to oversee a cluster or employ specialists in distributed systems.
+* Hardware configurations are optimized for each simulation (e.g., CPU vs. GPU, appropriate number of CPU cores, RAM, etc.).
```
Comment (sarmento) [inline README.md:15]: Automatically optimizes hardware configurations for each type of simulation

File: README.md [hunk 4]
```diff
@@ -3,11 +3,19 @@
...
+* You're not limited to a graphical interface or intricate configuration scripts. Instead, you can write concise Python programs that seamlessly integrate with your existing codebase.
```
Comment (sarmento) [inline README.md]: Move this to the first paragraph. This is the summary of what the API is.

**Review by fabiocruz — APPROVED** [review]

**Review by sarmento — APPROVED** [review]

**Comment by hpenedones** [comment]
@sarmento: will merge, but feel free to edit directly for some small changes that you might have in mind, if you notice something that can be improved. thanks!

**Pull request merged**